model/src/main/proto/spine_examples/shareaware/watchlist/commands.proto

```diff
@@ -50,3 +51,16 @@ message CreateWatchlist {
     // The name of the watchlist.
     string name = 3 [(required) = true];
 }
+
+// A wish of the user to watch for the share.
```

Comment:
We should explain the meaning, not just repeat the same verb. How about this:

"Tells that some user wants to observe changes in pricing for some share".